In [1]:
# %pip install langchain-openai python-dotenv

In [2]:
from dotenv import load_dotenv
load_dotenv()

True

In [3]:
from langchain_openai import ChatOpenAI

llm = ChatOpenAI()

In [6]:
llm.invoke("페리시치는 왤케 자살을 잘하지?")

AIMessage(content='페리시치가 자살을 잘하는 것은 그의 작품 속 인물들이 갖는 복잡한 심리와 갈등, 그리고 인간의 본성과 욕망을 다루기 때문일 것입니다. 또한 페리시치는 자살을 통해 인간의 존엄성, 불행과 비극을 다양한 각도에서 탐구하며 그것들을 표현하는 뛰어난 작가이기 때문일 것입니다. 결국, 그의 작품 속에서 자살은 단순한 죽음이 아닌 인간의 본질과 정체성을 깊이 있게 탐구하는 수단으로 사용되는 것으로 보입니다.', additional_kwargs={'refusal': None}, response_metadata={'token_usage': {'completion_tokens': 213, 'prompt_tokens': 27, 'total_tokens': 240, 'completion_tokens_details': {'accepted_prediction_tokens': 0, 'audio_tokens': 0, 'reasoning_tokens': 0, 'rejected_prediction_tokens': 0}, 'prompt_tokens_details': {'audio_tokens': 0, 'cached_tokens': 0}}, 'model_provider': 'openai', 'model_name': 'gpt-3.5-turbo-0125', 'system_fingerprint': None, 'id': 'chatcmpl-D2yl519oxeqJDmzppGMXdqHYs2Lym', 'service_tier': 'default', 'finish_reason': 'stop', 'logprobs': None}, id='lc_run--019c0488-9191-7fa3-834d-6ca14b7a1350-0', tool_calls=[], invalid_tool_calls=[], usage_metadata={'input_tokens': 27, 'output_tokens': 213, 'total_tokens': 240, 'input_token_details': {'aud

AIMessage(content='페리시치가 자살을 잘하는 것은 그의 작품 속 인물들이 갖는 복잡한 심리와 갈등, 그리고 인간의 본성과 욕망을 다루기 때문일 것입니다. 또한 페리시치는 자살을 통해 인간의 존엄성, 불행과 비극을 다양한 각도에서 탐구하며 그것들을 표현하는 뛰어난 작가이기 때문일 것입니다. 결국, 그의 작품 속에서 자살은 단순한 죽음이 아닌 인간의 본질과 정체성을 깊이 있게 탐구하는 수단으로 사용되는 것으로 보입니다.', additional_kwargs={'refusal': None}, response_metadata={'token_usage': {'completion_tokens': 213, 'prompt_tokens': 27, 'total_tokens': 240, 'completion_tokens_details': {'accepted_prediction_tokens': 0, 'audio_tokens': 0, 'reasoning_tokens': 0, 'rejected_prediction_tokens': 0}, 'prompt_tokens_details': {'audio_tokens': 0, 'cached_tokens': 0}}, 'model_provider': 'openai', 'model_name': 'gpt-3.5-turbo-0125', 'system_fingerprint': None, 'id': 'chatcmpl-D2yl519oxeqJDmzppGMXdqHYs2Lym', 'service_tier': 'default', 'finish_reason': 'stop', 'logprobs': None}, id='lc_run--019c0488-9191-7fa3-834d-6ca14b7a1350-0', tool_calls=[], invalid_tool_calls=[], usage_metadata={'input_tokens': 27, 'output_tokens': 213, 'total_tokens': 240, 'input_token_details': {'audio': 0, 'cache_read': 0}, 'output_token_details': {'audio': 0, 'reasoning': 0}})